src/JsonHelper.ts

```diff
@@ -1,17 +1,42 @@
-// @ts-nocheck
-import { Schema, SchemaHelper, ConfluentSubject, ConfluentSchema } from './@types'
+import {
+  Schema,
+  SchemaHelper,
+  ConfluentSubject,
+  ConfluentSchema,
+  SchemaResponse,
+  SchemaType,
+  ReferenceType,
+  JsonConfluentSchema,
+  ProtocolOptions,
+} from './@types'
 import { ConfluentSchemaRegistryError } from './errors'
 
 export default class JsonHelper implements SchemaHelper {
-  public validate(schema: Schema): void {
+  public validate(_schema: Schema): void {
```

Comment:
a little confusion with naming, in the interface `SchemaHelper` without underscore `validate(schema: Schema): void` - not critical but better to have consistency.